blackbox/monitoring/src/tests.py

```diff
@@ -230,25 +231,23 @@ def test_parent_breaker(self):
         jmx_client = JmxTermClient(JMX_PORT)
         stdout, stderr = jmx_client.query_jmx(
             'io.crate.monitoring:type=CircuitBreakers', 'Parent')
-        self.assertEqual(stdout, '{ \n'
-                                 '  limit = 726571417;\n'
-                                 '  name = parent;\n'
-                                 '  overhead = 1.0;\n'
-                                 '  trippedCount = 0;\n'
-                                 '  used = 0;\n'
-                                 ' }\n')
+        self.assert_valid_circuit_breaker_jmx_output('parent', stdout)
         self.assertEqual(stderr, '')
+
         stdout, stderr = jmx_client.query_jmx(
             'io.crate.monitoring:type=CircuitBreakers', 'Query')
-        self.assertEqual(stdout, '{ \n'
-                                 '  limit = 622775500;\n'
-                                 '  name = query;\n'
-                                 '  overhead = 1.09;\n'
-                                 '  trippedCount = 0;\n'
-                                 '  used = 0;\n'
-                                 ' }\n')
+        self.assert_valid_circuit_breaker_jmx_output('query', stdout)
         self.assertEqual(stderr, '')
 
+    def assert_valid_circuit_breaker_jmx_output(self, cb_name, output):
+        limit = re.search('limit = ([0-9]+);', output)
+        self.assertGreater(int(limit.group(1)), 0)
+
+        self.assertRegex(output, f'name = {cb_name};')
+        self.assertRegex(output, 'overhead = (\\d*\\.?\\d*);')
```

Comment:
I think we should replace `*` with `+` for all those digits here and next lines.